src/System.IO.Packaging/src/System/IO/Packaging/ZipWrappingStream.cs

```diff
@@ -88,25 +84,14 @@ public override void Flush()
             _baseStream.Flush();
         }
 
-        public override long Position
+        public override long Length
         {
             get
             {
-                return _baseStream.Position;
-            }
-            set
-            {
-                _baseStream.Position = value;
-            }
-        }
-
-        // For some strange reason, if the package FileAccess == Read,
-        // then can't get length from the stream.  Not supported.
-        // The workaround - write the stream to a memory stream, then return the length of the
-        // memory stream.
-        public override long Length
-        {
-            get {
+                // For some strange reason, if the package FileAccess == Read,
+                // then can't get length from the stream.  Not supported.
+                // The workaround - write the stream to a memory stream, then return the length of the
+                // memory stream.
                 if (_packageFileAccess == FileAccess.Read)
```

Comment:
That wouldn't be the length of the final decompressed bytes though, it would be the length of the compressed stream.

It's specific enough that it is fine as is.